src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -170,6 +195,24 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    /**
+     * Display and refresh date time display on TutorBuddy every 1 second
+     */
+    private void displayDateAndTime() {
```

Comment:
Hi, was just thinking that maybe displaying date and then time could be better in this case, just a suggestion ah as it feels more intuitive for me, although is debatable.

Reply:
Can idm also!